src/backend/catalog/gp_warehouse.c

```diff
@@ -0,0 +1,73 @@
+#include "postgres.h"
+
+#include "access/htup_details.h"
+#include "access/genam.h"
+#include "access/table.h"
+#include "catalog/catalog.h"
+#include "catalog/indexing.h"
+#include "catalog/gp_warehouse.h"
+
+#include "utils/builtins.h"
+#include "utils/rel.h"
+
+void
+GpWarehouseCreate(char *warehouse_name)
```

Comment:
We don't need to implement GpWarehouseCreate in CBDB?

Reply:
I have move it to extension